document/initialization.md

```diff
@@ -12,8 +12,19 @@
 - `$ wget https://dlcdn.apache.org/maven/maven-3/3.8.8/binaries/apache-maven-3.8.8-bin.tar.gz`
 - `$ tar -xvf apache-maven-3.8.8-bin.tar.gz`
 
+- install check `$ java -version`
+
 ### Install MySQL
 - `$ sudo dnf install mysql mysql-server`
+- `$ sudo systemctl enable mysqld` (for auto restart)
+- active check `$ sudo systemctl status mysqld`
 
 ### Install Redis
 - `$ sudo dnf install redis`
+- active check 
+    - `$ sudo cp /usr/lib/systemd/system/redis.service /etc/systemd/system/redis.service`
+    - `$ sudo systemctl daemon-reload`
+    - `$ sudo systemctl enable redis` (for auto restart)
+    - `$ sudo systemctl start redis`
+    - `$ sudo systemctl status redis`
```

Comment:
```
$ sudo systemctl status redis
● redis.service - Redis persistent key-value database
     Loaded: loaded (/etc/systemd/system/redis.service; enabled; preset: disabled)
    Drop-In: /etc/systemd/system/redis.service.d
             └─limit.conf
     Active: active (running) since Sat 2024-07-20 21:27:19 KST; 10min ago
   Main PID: 791 (redis-server)
     Status: "Ready to accept connections"
      Tasks: 5 (limit: 10704)
     Memory: 9.6M
        CPU: 690ms
     CGroup: /system.slice/redis.service
             └─791 "/usr/bin/redis-server 127.0.0.1:6379"

Jul 20 21:27:19 ip-172-31-41-61.ap-southeast-2.compute.internal systemd[1]: Starting R>
Jul 20 21:27:19 ip-172-31-41-61.ap-southeast-2.compute.internal systemd[1]: Started Re>
lines 1-15/15 (END)
```